recipes-core/systemd/systemd_%.bbappend

```diff
@@ -11,5 +15,11 @@ do_install_append() {
 
 FILES_${PN} += "/etc/resolv.conf"
 FILES_${PN} += "${sysconfdir}/systemd/system/getty.target.wants"
+FILES_${PN} += "/lib/sysctl.d/99-printk.conf"
```

Comment:
${exec_prefix}/lib/sysctl.d ? Will expand to /usr/lib/sysctl.d, I think. What the systemd recipe uses:

http://cgit.openembedded.org/openembedded-core/tree/meta/recipes-core/systemd/systemd_239.bb?h=thud#n540

Actually, I do not see a /lib/sysctl.d directory on target. Have you tested that this actually works? Regardless, using variables should give you correct location...?

Reply:
so, where should this be deployed, `/lib` or `/usr/lib` ?